### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by katarzyna-z**
Summary of changes:
- Updated examples (removed usage of mock plugins),
- Snap capitalized
- Changed path to plugin binary path
- Changed supported golang version (adjust to travis tests)

How to verify it:
- do steps describted in example
- read documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,70 +113,75 @@ $ docker run --name heka -it -p 4352:4352 -p 3242:3242 -v <path to heka-tcp-conf
...
+
+Download and load snap-plugin-collector-psutil plugin (path to binary file for Linux/amd64):
 ```
-Load snap heka publisher plugin
+$ wget http://snap.ci.snap-telemetry.io/plugins/snap-plugin-collector-psutil/latest/linux/x86_64/snap-plugin-collector-psutil
+$ snapctl plugin load snap-plugin-collector-psutil
 ```
-$ $SNAP_PATH/bin/snapctl plugin load <path to built snap-plugin-publisher-heka plugin>
+
+Build heka according to the [instruction](#building-from-source) and go to directory with plugin binary file.
```
Comment (andrzej-k) [inline README.md]: Consider also adding wget to heka plugin - just in case user just wants to try or use the plugin but not necessarily build the plugin.

Comment (katarzyna-z) [inline README.md]: It seems that currently binary file of heka plugin is not available. Trying to get the binary file with this command `wget http://snap.ci.snap-telemetry.io/plugins/snap-plugin-publisher-heka/latest/linux/x86_64/snap-plugin-publisher-heka` I receive`404 Not Found` error. The [release page](https://github.com/intelsdi-x/snap-plugin-publisher-heka/releases) is also empty.

Comment (andrzej-k) [inline README.md]: Ok, so let's address releasing this plugin in separate issue.

Comment (katarzyna-z) [inline README.md]: I've opened #25

Comment (katarzyna-z) [inline README.md]: I've opened #25

Comment (katarzyna-z) [inline README.md]: I've opened #25

**Pull request merged**